### PR TITLE
Added points to map, playing around with an infowindow

### PR DESCRIPTION
**Pull request by ttlarson7**

### CODE REVIEW

File: src/components/explore.tsx [hunk 1]
```diff
@@ -90,44 +107,65 @@ const studyRooms = [
...
 
 export function ExplorePage({ className, ...props }: CardProps) {
   const [data, setData] = useState<GeoJSON | null>(null);
-
+  const [testData, setTestData] = useState<StudyRoom[]>(studyRooms);
   const navigate = useNavigate();
 
+  const contentString = "<div>KEC 1001 Study Room</div>";
+  
+
```
Comment (sebat2004) [inline src/components/explore.tsx:152]: You declared this but didn't end up using it

**Review by sebat2004 — CHANGES_REQUESTED** [review]
One small fix but the rest LGTM

**Review by PotatoPresident — APPROVED** [review]
LGTM

File: src/components/explore.tsx (hunk 1 above)
Comment (sebat2004) [reply src/components/explore.tsx:152]: Ah shucks who cares it looks good to Will

**Review by sebat2004 — APPROVED** [review]

**Pull request merged**